### PR TITLE
fix issues in translation file

### PR DESCRIPTION
**Pull request by denyskon**
- use correct comment sign for INI (`;`)
- remove duplicated `repo.branch.search` key
- remove duplicated spaces & similar

### CODE REVIEW

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -180,7 +180,7 @@ occurred = An error occurred
 report_message = If you believe that this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
 missing_csrf = Bad Request: no CSRF token present
 invalid_csrf = Bad Request: invalid CSRF token
-not_found = The target couldn't be found.
+not_found = "The target couldn't be found."
```
Comment (delvh) [inline options/locale/locale_en-US.ini]: @wxiaoguang wasn't there something about quotes being bad for Crowdin?

Comment (denyskon) [reply options/locale/locale_en-US.ini]: They're being used in many strings already.....

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini]: The only problem is something like `key = "the quoted string" is at the beginning`

https://github.com/go-gitea/gitea/blob/main/modules/translation/i18n/i18n_test.go#L120C1-L120C93

File: options/locale/locale_en-US.ini [hunk 2]
```diff
@@ -192,7 +192,7 @@ platform_desc = Gitea runs anywhere <a target="_blank" rel="noopener noreferrer"
 lightweight = Lightweight
 lightweight_desc = Gitea has low minimal requirements and can run on an inexpensive Raspberry Pi. Save your machine energy!
 license = Open Source
-license_desc = Go get <a target="_blank" rel="noopener noreferrer" href="https://code.gitea.io/gitea">code.gitea.io/gitea</a>! Join us by <a target="_blank" rel="noopener noreferrer" href="https://github.com/go-gitea/gitea">contributing</a> to make this project even better. Don't be shy to be a contributor!
+license_desc = "Go get <a target="_blank" rel="noopener noreferrer" href="https://code.gitea.io/gitea">code.gitea.io/gitea</a>! Join us by <a target="_blank" rel="noopener noreferrer" href="https://github.com/go-gitea/gitea">contributing</a> to make this project even better. Don't be shy to be a contributor!"
```
Comment (lunny) [inline options/locale/locale_en-US.ini]: use ` ?

Comment (wxiaoguang) [reply options/locale/locale_en-US.ini]: No, don't use it. It has already caused many issues.

**Comment by wxiaoguang** [comment]
> add quotes around translation where needed

I think most (or all) of them are not needed.

File: options/locale/locale_en-US.ini (hunk 2 above)
Comment (lunny) [reply options/locale/locale_en-US.ini]: So maybe this change is unnecessary.

**Comment by silverwind** [comment]
We need a better format than ini like yaml where such bikeshedding around quotes is automatically handled by a linter 😆.

**Comment by 6543** [comment]
best would be to write testcases to lint the locales file!!!!

could you write some tests for that?

**Comment by silverwind** [comment]
I already checked and there are no usable linters for ini files around. We shouldn't invest too much energy in it, just move to a better format.

**Review by delvh — APPROVED** [review]
LGTM, hoping that it won't break anything

**Comment by wxiaoguang** [comment]
> LGTM, hoping that it won't break anything

some changes like `gpg_helper = "<strong>Need help?</strong> Have a look at GitHub's guide <a href="%s">about GPG</a>."` doesn't seem right. The quote appears in other quotes

**Comment by denyskon** [comment]
Seems like the consensus is that quotes rather introduce more problems, so I'll remove them again

**Comment by denyskon** [comment]
I reverted the quote changes

**Review by wxiaoguang — APPROVED** [review]

**Pull request merged**